neo4j-cli/aura/internal/subcommands/graphanalytics/session/list.go

```diff
@@ -0,0 +1,62 @@
+package session
+
+import (
+	"net/http"
+
+	"github.com/neo4j/cli/common/clicfg"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/api"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/output"
+	"github.com/spf13/cobra"
+)
+
+func NewListCmd(cfg *clicfg.Config) *cobra.Command {
+	var tenantId string
+	var instanceId string
+	var organizationId string
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "Returns a list of Graph Analytics Serverless sessions",
+		Long: `This subcommand returns a list containing a summary of each of your Graph Analytics Serverless session
+				By default, this subcommand lists all sessions a user has access to across all projects.
+				You can filter sessions in a particular project/tenant using:
+				--organization-id <organization-id>
+				--tenant-id <tenant-id>
+				--instance-id <instance-id>
+`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			path := "/graph-analytics/sessions"
+
+			queryParams := make(map[string]string)
+			if organizationId != "" {
+				queryParams["organizationId"] = organizationId
+			}
+			if tenantId != "" {
+				queryParams["tenantId"] = tenantId
```

Comment:
You need to be mapping the `--tenant-id` flag over to `projectId` right? Does your API endpoint still only accept `projectId`? 

```suggestion
				queryParams["projectId"] = tenantId
```

I won't comment this in all places, just here, but this would obviously need reflecting in all commands.

Additionally, you haven't included the logic for the default tenant ID here. 🙂

Reply:
we also accept tenantId now as a parameter.

Just checking not both `projectId` and `tenantId` is given.

Will look into adding the logic for the default tenant ID 🤞

Reply:
checking `instances/list.go` the default tenant logic is also not present? Was this on purpose or is this a bug for list instances?

Reply:
i think for listing the issue is `how do you list over all tenants`. 
You will need to specify all tenants present as its not the default if we use `default tenant` as the default for the filter @darrellwarde